models/User.go

```diff
@@ -740,7 +740,19 @@ func PlaceAskOrder(userId uint32, ask *Ask) (uint32, error) {
 		return 0, NotEnoughStocksError{currentAllowedQty}
 	}
 
-	l.Debugf("Check2: Passed. Creating Ask.")
+	l.Debugf("Check2: Passed.")
+
+	orderFee := uint32((ORDER_FEE_PERCENT / 100.0) * float32(ask.StockQuantity*ask.Price))
+	cashLeft := uint32(user.Cash) - orderFee
```

Comment:
Don't forget to change these to uint64 in a later PR since we're updating that now. Looks like you will have to change ```User.Cash``` to uint64 as well.

Reply:
Also, `user.Cash` is already `uint32` so `uint32(user.Cash)` is pointless

Reply:
This actually would have fucked up. Like because both are unsigned it would never become less than MINIMUM_CASH_LIMIT which is 0. So in the case where cash = 0 , order fee = 10, uint32(0 - 10) would be 0 which would pass the test.

Reply:
why didn't the test catch it then?

Reply:
Well it sort of failed when I changed it. Thats how I caught it.